### PR TITLE
Release Version 1.2.0

### PR DESCRIPTION
**Pull request by oxzi**
Part of #707. Should be merged last before drafting the release.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,29 @@
...
+with a particular focus on retryable database errors that may occur when using Icinga DB with database clusters.
+
+* Retry broken pipe errors (`EPIPE`). #693
+* Retry every database error. #698
+* MySQL/MariaDB: Use strict SQL mode. #699
+* Try setting `wsrep_sync_wait` for MySQL/MariaDB connections. #711
+* history.Sync: Don't operate on closed channel. #725
+* Flatten(): render even large numbers as-is, not using scientific notation. #657
+* Enhance HA "Taking over", "Handing over" logging. #692
+* Update dependencies. #649 #650 #658 #661 #662 #663 #664 #670 #673 #675 #678 #680 #682 #687 #696 #701 #702 #703 #704 #705 #706 #712 #715 #720 #721 #727 #728 #733
```
Comment (julianbrost) [inline CHANGELOG.md:15]: It's done like last time, but that PR list is very long and doesn't look helpful at all. Maybe just link to https://github.com/Icinga/icingadb/pulls?q=is%3Apr+milestone%3A1.1.2+label%3Adependencies instead?

File: CHANGELOG.md
Comment (julianbrost) [inline CHANGELOG.md:1]: I'd stay there's still some improvement possible regarding understandability for end users, i.e. saying that `wsrep_sync_wait` is for Galera clusters, `Flatten()` for custom variables, etc.

Comment (oxzi) [reply CHANGELOG.md:1]: I added initial prose for essential changes and reworded some of the bullet points to make the change's impact more obvious.

**Review by julianbrost — COMMENTED** [review]
Should be fine, I'm waiting for everything else to be done for a final approval though.

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,37 @@
...
+The latter was specifically addressed in #711 for Galera clusters on MySQL or MariaDB by configuring `wsrep_sync_wait` on used database sessions.
+Galera users should refer to the documentation in `doc/03-Configuration.md` for more details.
+
+* Custom Variables: Render large numbers as-is, not using scientific notation. #657
+* Enhance HA "Taking over", "Handing over" logging. #692
+* Retry broken pipe errors (`EPIPE`). #693
+* Retry every database error. #698
+* MySQL/MariaDB: Use strict SQL mode. #699
+* MySQL/MariaDB Galera Cluster: Try setting `wsrep_sync_wait`. #711
+* Don't crash history sync in the absence of Redis. #725
```
Comment (julianbrost) [inline CHANGELOG.md:22]: Maybe this should also get a funny little R, see #736.

Comment (oxzi) [reply CHANGELOG.md]: Added the funny little ᴙ and linked to the trademark document.

<sub><sup>~~As their trademark policy is not RFC 2119 compliant, section 5.a i. could be ignored.~~</sup></sub>

**Review by julianbrost — CHANGES_REQUESTED** [review]
The PR has to be adapted to the short notice PRs. At least #739 should be mentioned as `EPIPE` is mentioned explicitly already. (There might be more noteworthy ones, I haven't checked all so far.)

**Comment by oxzi** [comment]
@julianbrost: I rebased this PR's branch onto main. Afterwards, I reworded the CHANGELOG to group similar changes (especially everything retry-related) and include the latest additions.

**Review by yhabteab — APPROVED** [review]

**Comment by julianbrost** [comment]
> Afterwards, I reworded the CHANGELOG to group similar changes (especially everything retry-related) and include the latest additions.

Looks like you have lost #692 in the process.

**Comment by oxzi** [comment]
> Looks like you have lost #692 in the process.

This was intentional, as, on second thought, I found this change to be irrelevant for a CHANGELOG entry.

**Review by julianbrost — APPROVED** [review]

**Review by lippserd — APPROVED** [review]

**Pull request merged**